README.md

```diff
@@ -43,6 +43,12 @@ libpcap should be installed on Mac OS X by default.
 
 If `PCAP_LIBDIR` environment variable is set when building the crate, it will be added to the linker search path - this allows linking against a specific `libpcap`.
 
+## Library Version
+
+The crate will automatically try to detect the installed `libpcap`/`wpcap` version by loading it during the build and calling `pcap_lib_version`. If for some reason this is not suitable, you can specify the desired library version by setting the environment variable `PCAP_VER` to the desired version. The version number is used to determine which library calls to include in the compilation.
```

Comment:
Change to `LIBPCAP_VER` to avoid mix up with the crate version?

Reply:
Done